mitxgraders/helpers/calc/expressions.py

```diff
@@ -586,7 +590,22 @@ def __init__(self, expression, tree, variables_used, functions_used, suffixes_us
         self.tree = tree
 
     def __str__(self):
-        return self.tree.asXML()
+        tree_dict = self.tree.as_dict()
```

Comment:
To be quite honest, these functions were here for debugging purposes only, and could be removed completely. I'd actually suggest commenting them out so that the xml library doesn't need to be loaded, because the grader is on a short timeout, and loading that library isn't worth it.

Reply:
So, we want to comment the `str` method and edit this doc test, right? To be able to avoid the import of xml librery.

Test: https://github.com/mitodl/mitx-grading-library/blob/065bc457f622a38fb8cea14449bdddb439be12ca/mitxgraders/helpers/calc/expressions.py#L312

Reply:
I'll comment it.